v2/api/containerservice/v1alpha1api20210501/managed_cluster_types_gen.go

```diff
@@ -9917,18 +9917,18 @@ func (profile *ManagedClusterLoadBalancerProfile) AssignProperties_From_ManagedC
 
 	// EffectiveOutboundIPs
 	if source.EffectiveOutboundIPs != nil {
-		effectiveOutboundIPList := make([]ResourceReference, len(source.EffectiveOutboundIPs))
-		for effectiveOutboundIPIndex, effectiveOutboundIPItem := range source.EffectiveOutboundIPs {
+		effectiveOutboundIpList := make([]ResourceReference, len(source.EffectiveOutboundIPs))
```

Comment:
minor: the previous casing was actually slightly more correct (for Go).

It's not a big deal though if this is hard to fix.